include/unicore-mx/usbd/usbd.h

```diff
@@ -279,7 +284,7 @@ enum usbd_transfer_flags {
 	USBD_FLAG_PACKET_PER_FRAME_MASK = (0x3 << 5)
 };
 
-typedef enum usbd_transfer_flags usbd_transfer_flags;
```

Comment:
yep, i know about namespaces, and that code was compiled succesfully.
when i ask about "valid practice for C" - i mead it it good? is it helpful? can it lead to some errors?
the name "usbd_transfer_flags" imho to very suitable for enum. such name also can be imagine for struct of a binary set of flags.
anyway, if you insist - let`s revert this typedef back to code.